lib/experimental/new_embed.dart

```diff
@@ -121,7 +117,8 @@ String stringify(int x, int y) {
 }
 ''';
 
-  void _handleRun() {
+  void _handleExecute() {
+    executeButton.ready = false;
```

Comment:
You might instead make the button disabled; I think just adding a `disabled` attribute should do it.

It would look like:

<img width="245" alt="screen shot 2019-03-01 at 3 25 31 pm" src="https://user-images.githubusercontent.com/1269969/53672222-6b715200-3c36-11e9-9335-f3c2fccd5f11.png">

Reply:
> @jacob314, I'm importing Primer.css and Octicons using the packages you created for DevTools:
> 
> https://github.com/flutter/devtools/tree/master/third_party/packages
> 
> If that's a terrible idea, let me know.

That is a completely reasonable idea. The packages don't currently have any actually dart specific content. It could make sense to add some dart specific content to the packages assuming licensing concerns are addressed.

It will be great to have a consistent look and feel across the various Dart tools.

Reply:
Turned out to be a disabled CSS class because I'm using anchor elements instead of button elements, which is, I now realize, an old habit. I'll probably switch it to be a real button in a subsequent PR. Thanks for the tip.

Reply:
@jacob314 I'm glad reusing the package isn't a bad idea, because it's quite convenient for me. 😄 

One question: it looks like the version of Octicons in the package (4.4.0) is significantly behind the current (8.4.2). Is that WAI, or could I submit an update to the package?

Reply:
Feel free to update the Octicons package. We just haven't updated because we haven't needed to.  I expect you will be able to update without causing any issues in the devtools app. I was able to update the version of primer.css package to the latest version a fews ago and only one style needed to be updated in our code. Expect updating Octicons should be even less risky.

One warning: we don't yet have golden image tests of the UI so you won't be able to automatically test whether updated Octicons package caused any issues.
I would suggest running the latest version of the devtools app side by side with the version with the updated icons to make sure nothing changed in a a surprising way.